packages/cli/src/models/network/NetworkController.ts

```diff
@@ -105,179 +115,176 @@ export default class NetworkController {
+  // Contract model || SolidityLib model
+  private async _unsetSolidityLibs(): Promise<void> {
+    const { contractNames } = this.packageFile;
+    const libNames = this._getAllSolidityLibNames(contractNames);
     await allPromisesOrError(
-      this.networkFile.solidityLibsMissing(libNames).map(libName => this._unsetSolidityLib(libName))
-    )
+      this.networkFile.solidityLibsMissing(libNames).map((libName) => this._unsetSolidityLib(libName))
+    );
   }
 
-  //Contract model || SolidityLib model
-  async _unsetSolidityLib(libName) {
+  // Contract model || SolidityLib model
+  private async _unsetSolidityLib(libName: string): Promise<void | never> {
     try {
       log.info(`Removing ${libName} library`);
       await this.project.unsetImplementation(libName);
-      this.networkFile.unsetSolidityLib(libName)
+      this.networkFile.unsetSolidityLib(libName);
     } catch(error) {
-      error.message = `Removal of ${libName} failed with error: ${error.message}`
-      throw error
+      error.message = `Removal of ${libName} failed with error: ${error.message}`;
+      throw error;
     }
   }
 
-  //Contract model || SolidityLib model
-  _hasChangedLibraries(contractClass, changedLibraries) {
-    const libNames = getSolidityLibNames(contractClass.bytecode)
-    return !_.isEmpty(_.intersection(changedLibraries.map(c => c.contractName), libNames))
+  // Contract model || SolidityLib model
+  private _hasChangedLibraries(contractClass: ContractFactory, changedLibraries: ContractFactory[]): boolean {
+    const libNames = getSolidityLibNames(contractClass.bytecode);
+    return !_.isEmpty(_.intersection(changedLibraries.map((c) => c.contractName), libNames));
   }
 
-  //Contract model || SolidityLib model
-  _getAllSolidityLibNames(contractNames) {
-    const libNames = contractNames.map(contractName => {
-      const contractClass = Contracts.getFromLocal(contractName)
-      return getSolidityLibNames(contractClass.bytecode)
-    })
+  // Contract model || SolidityLib model
+  private _getAllSolidityLibNames(contractNames: string[]): string[] {
+    const libNames = contractNames.map((contractName) => {
+      const contractClass = Contracts.getFromLocal(contractName);
+      return getSolidityLibNames(contractClass.bytecode);
+    });
 
-    return _.uniq(_.flatten(libNames))
+    return _.uniq(_.flatten(libNames));
   }
 
-  //Contract model
-  async unsetContracts() {
+  // Contract model
+  public async unsetContracts(): Promise<void> {
     await allPromisesOrError(
-      this.networkFile.contractAliasesMissingFromPackage().map(contractAlias => this.unsetContract(contractAlias))
-    )
+      this.networkFile.contractAliasesMissingFromPackage().map((contractAlias) => this.unsetContract(contractAlias))
+    );
   }
 
-  //Contract model
-  async unsetContract(contractAlias) {
+  // Contract model
+  public async unsetContract(contractAlias: string): Promise<void | never> {
```

Comment:
nit: private.

Reply:
Is it private because you know it should be used that way or because of something else? So far I've been using private only for Js function that start with an underscore.

Reply:
That's great. However, I think we should worry about encapsulation at a later stage of the migration.